windowing.py

```diff
@@ -205,3 +205,84 @@ def cropped_np(np_sample, start, end):
     end_index = np.where(nanos == end)
     cropped = [np_sample[:,  start_index[0][0]:end_index[0][0] + 1]]
     return cropped
+
+
+def place_data(np_sample, data_key, data_label, dictionary, positive, window_size, stride):
+    """
+        Description: windows the np_sample (np array) with desired specifications and then
+                     places the windows into dictionary["data_key"] and creates a
+                     corresponding number of labels to dictionary["data_label"]
+
+        Args:
+            np_sample -- np array
+            data_key -- String (key within dictionary to place windows: dictionary[data_key])
+            data_label -- String (key within dictionato place label: dictionary[data_label])
+            dictionary -- dict (dictionary where windows and labels will be added to)
+            positive -- boolean (binary label of windows)
+            window_size -- int
+            stride -- int (# of samples to jump before creating next window)
+
+
+        Return:
+            dictionary -- dict (updated dictionary with added windows and labels)
+        """
+    rows = len(np_sample)
+    windowed_data = view_as_windows(np_sample, (rows, window_size), stride)
+    dictionary[data_key].extend(windowed_data[0])
+    if (positive):
+        pos_or_neg = np.ones(len(windowed_data[0]))
+    else:
+        pos_or_neg = np.zeros(len(windowed_data[0]))
+    dictionary[data_label].extend(pos_or_neg)
+    return dictionary
+
+
+def choose_dataset(data_count, num_test, num_train, num_validation):
```

Comment:
Based on the place_data function above, you should have a list of indices into the data_label and data_key parts of the dictionary. Could you split them up among train, validation, and test by splitting up the indicies? I assume you also have a function to get them ready to be passed into TensorFlow so maybe the choose_dataset function is better setup for that. And would you need to decide the number of train, test, and validation beforehand? I think adding percentages seems like an easier way to me so you don't have to worry about computing those on the fly and already knowing the dataset size.